lib/ex_rabbit_m_q/consumer.ex

```diff
@@ -475,7 +493,8 @@ defmodule ExRabbitMQ.Consumer do
           queue: config[:queue],
           queue_opts: config[:queue_opts],
           consume_opts: config[:consume_opts],
-          bind_opts: config[:bind_opts],
+          exchange_opts: config[:exchange_opts],
+          bind_opts: config[:bind_opts]
```

Comment:
should we pull extra_opts out of bind_opts?

Reply:
Ok, done. `bind_opts` now contain only the `options` for the `AMQP.Queue.Bind` function.